packages/flutter_tools/lib/src/ios/native_assets.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:native_assets_builder/native_assets_builder.dart' show BuildResult;
+import 'package:native_assets_cli/native_assets_cli.dart' hide BuildMode;
+import 'package:native_assets_cli/native_assets_cli.dart' as native_assets_cli;
+
+import '../base/file_system.dart';
+import '../build_info.dart';
+import '../globals.dart' as globals;
+
+import '../macos/native_assets_host.dart';
+import '../native_assets.dart';
+
+/// Dry run the native builds.
+///
+/// This does not build native assets, it only simulates what the final paths
+/// of all assets will be so that this can be embedded in the kernel file and
+/// the Xcode project.
+Future<Uri?> dryRunNativeAssetsIOS({
+  required NativeAssetsBuildRunner buildRunner,
+  required Uri projectUri,
+  required FileSystem fileSystem,
+}) async {
+  if (await hasNoPackageConfig(buildRunner) || await isDisabledAndNoNativeAssets(buildRunner)) {
+    return null;
+  }
+
+  final Uri buildUri_ = nativeAssetsBuildUri(projectUri, OS.iOS);
+  final Iterable<Asset> assetTargetLocations = await dryRunNativeAssetsIOSInternal(fileSystem, projectUri, buildRunner);
```

Comment:
Formatting: [Use a trailing comma for arguments, parameters, and list items, but only if they each have their own line... [if] everything would fit on one line, put it all on one line, otherwise, split it one item to a line.](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line)

So for example, since this line doesn't all fit on one line within 80 characters, use a trailing command and give each item its own line.

```dart
final Iterable<Asset> assetTargetLocations = await dryRunNativeAssetsIOSInternal(
  fileSystem, 
  projectUri, 
  buildRunner,
);
```

There's a couple of other places you can do this in this file